<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>215</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="resultLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>результат:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="resultEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="inputLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>введите значение:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>234</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="numberafterpoint">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>количество знаков после .</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="calculateLayout">
       <item>
        <widget class="QSpinBox" name="accuracySpinBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="calculateButton">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
